### DISCUSSION TITLE
Ignore gh_pages (or really any branch) in the repo config?

### DISCUSSION THREAD

**Discussion by wez** [woodpecker-ci/woodpecker#3120] [Ask a Question] [answered]
I'm bothered that this shows as an error each time I have a gh_pages commit:

![image](https://github.com/woodpecker-ci/woodpecker/assets/117777/373ffec2-6da7-4e76-8e4a-0840f46b80a1)

(Also: the "not started yet" implies that it is going to start, and that makes me feel extra bothered!)

There is no and never will be any pipeline config for this branch.

How can I make woodpecker ignore it?

**Comment by qwerty287** [comment, marked as answer]
Hasn't this been fixed already?

https://github.com/woodpecker-ci/woodpecker/pull/2949

**Reply by qwerty287** [reply]
This should be included in 2.1.0. Which version are you using?

**Reply by wez** [reply]
ah, good to hear! I was on 2.0 and just updated to 2.1.1 today. I haven't pushed any docs changes yet, but I look forward to not seeing anything about them in the UI :)
